doc/specifications/identities/Remote-identities-creation-diagram.puml

```diff
@@ -0,0 +1,58 @@
+@startuml
+actor "Authenticated user" as user
+participant "Client" as client
+database "Backend" as backend
+user -->o client: click\n« add remote account »
```

Comment:
`click "settings > external accounts"`

Actually the user lists the remote identities and can directly create on one provider